### ISSUE TITLE
__runOriginal in Postfix always return false

### ISSUE THREAD

**Issue by BarehSolok** [pardeike/Harmony#473]
**Describe the bug**
__runOriginal in Postfix always return false even if original was not skipped.

**Expected behavior**
__runOriginal in Postfix should return true if original method was not skipped.

**Screenshots / Code**
```cs
[HarmonyPatch(typeof(ZombieManager), "sendZombieBoulder")]
[HarmonyPrefix]
internal static bool sendZombieBoulder(bool __runOriginal, Zombie zombie, Vector3 origin, Vector3 direction)
{
    Console.WriteLine($"[DEBUG] Prefix __runOriginal:{__runOriginal}");
    return true;
}

[HarmonyPatch(typeof(ZombieManager), "sendZombieBoulder")]
[HarmonyPostfix]
internal static void sendZombieBoulderPost(bool __runOriginal, Zombie zombie, Vector3 origin, Vector3 direction)
{
    Console.WriteLine($"[DEBUG] Postfix __runOriginal:{__runOriginal}");
}
```
![image](https://user-images.githubusercontent.com/83990509/164939389-14d198d5-9dfc-4076-b356-57c473ea7643.png)

**Runtime environment (please complete the following information):**
 - OS: Windows 10, 64bit
 - .NET version 4.8
 - Harmony version 2.2.1.0
- Name of game or host application Unturned

**Comment by pardeike** [comment]
I added at test for this in f99da8364dbbf0f0530e98fd070314d6bb81a9e3 and it looks like the code behaves correctly. Not sure why it fails for you.

**Comment by BarehSolok** [comment]
i can reproduce this with .Net framework 4.8 console app

```internal class Program
    {
        public static void Main(string[] args)
        {
            var harmony = new Harmony("test");
            harmony.PatchAll();
            
            MyClass.Test("asd");
        }
        
        public static class MyClass
        {
            public static void Test(string s)
            {
                Console.WriteLine("[DEBUG] Test method ran, Postfix __runOriginal should be True");
            }
        }
        
        [HarmonyPatch(typeof(MyClass), "Test")]
        public static class MyClass2
        {
            public static bool Prefix(bool __runOriginal, string s)
            {
                var pre = __runOriginal;
                Console.WriteLine($"[DEBUG] Prefix __runOriginal:{pre}");
                return true;
            }
            public static void Postfix(bool __runOriginal, string s)
            {
                var post = __runOriginal;
                Console.WriteLine($"[DEBUG] Postfix __runOriginal:{post}");
            }
        }
    }}```
![image](https://user-images.githubusercontent.com/83990509/164954644-56ceb961-9a40-4905-99cb-f6838a7340fa.png)

**Comment by pardeike** [comment]
After investigating a bit more, I found the reason we differ is that I already fixed it in this commit 6006140d929565cb49ee07d482f96ec0d51a7aed which is planed to be released in v2.2.2
